src/CronJobListBuilder.php

```diff
@@ -37,15 +38,22 @@ public function buildHeader() {
    */
   public function buildRow(EntityInterface $entity) {
     /* @var \Drupal\ultimate_cron\Entity\CronJob $entity */
+    $entry = $entity->loadLatestLogEntry();
+    // Start and end as UNIX timestamps.
+    $start_time = $entry->start_time;
+    $end_time = $entry->end_time;
+    // Duration in milliseconds.
+    $duration = ($end_time - $start_time) * 1000;
+    $duration = floor($duration);
```

Comment:
wondering if that is really a good way to display it, I'd suggest something like (0.01s). Also, displaying it below should use $this->t() with a placeholder.